docs/getting-started-with-sylius/deployment.rst

```diff
@@ -3,10 +3,14 @@ Deployment
 
 Development usually takes most of the time in project implementation, but we should not forget about what's at the end of this process -
 application deployment into the server. We believe, that it should be as easy and understandable as possible.
-There are many servers which you can choose for your store deployment: in our documentation you will find an easy Platform.sh guide.
 
-Check it out!
+Check out our deployment cook books:
+
+* :doc:`How to deploy Sylius to Artifakt? </cookbook/deployment/artifakt>`
+* :doc:`How to deploy Sylius to Cloudways? </cookbook/deployment/cloudways>`
+* :doc:`How to deploy Sylius to Platform.sh? </cookbook/deployment/platform-sh>`
+* :doc:`How to deploy Sylius to Symfony Cloud? </cookbook/deployment/symfonycloud>`
 
 .. tip::
 
-    👉 :doc:`How to deploy Sylius to Platform.sh? </cookbook/deployment/platform-sh>`
+    🐳 :doc:`How to deploy Sylius with Docker </cookbook/deployment/docker>`
```

Comment:
Please wait for https://github.com/Sylius/Sylius/pull/14023 to be merged and then add Docker to the list.